test/driver_turso_async/sqlc.yaml

```
@@ -0,0 +1,175 @@
+          emit_classes: false
+          omit_unused_models: true
+          emit_init_file: true
+          docstrings: google
+
+          overrides:
+            - column: test_type_override.text_test
+              py_type:
+                import: collections
+                package: UserString
+                type: UserString
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)